chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -80,10 +80,10 @@
 default[:service][:file] = "OpenStackService.exe"
 default[:service][:nova][:name] = "nova-compute"
 default[:service][:nova][:displayname] = "Openstack Nova Compute Service"
-default[:service][:nova][:description] = "Service Wrapper for Openstack Nova Compute - Manages the Openstack Nova Compute as a Windows Service"
+default[:service][:nova][:description] = "Service Wrapper for Openstack Nova Compute"
 default[:service][:neutron][:name] = "neutron-hyperv-agent"
 default[:service][:neutron][:displayname] = "OpenStack Neutron Hyper-V Agent Service"
-default[:service][:neutron][:description] = "Service Wrapper for Openstack Neutron Hyper-V Agent - Manages the OpenStack Neutron Hyper-V Agent as a Windows Service"
-default[:service][:ceilometer][:name] = "ceilometer-compute-agent"
-default[:service][:ceilometer][:displayname] = "OpenStack Ceilometer Compute Agent Service"
-default[:service][:ceilometer][:description] = "Service Wrapper for Openstack Ceilometer Compute Agent - Manages the OpenStack Ceilometer Compute Agent as a Windows Service"
+default[:service][:neutron][:description] = "Service Wrapper for Openstack Neutron Hyper-V Agent"
+default[:service][:ceilometer][:name] = "ceilometer-agent-notification"
```

Comment:
because of the 100 characters limit hound is bitching about. Have a chat with @tboerger about that.